config/schema/elasticsearch_types/research_for_development_output.json

```diff
@@ -0,0 +1,956 @@
+{
+  "fields": [
+    "country",
+    "authors",
+    "research_document_type",
+    "review_status",
+    "theme",
+    "business_activity",
+    "employ_eu_citizens",
+    "first_published_at",
+    "intellectual_property",
+    "personal_data",
+    "public_sector_procurement",
+    "eu_uk_government_funding",
+    "regulations_and_standards",
+    "sector_business_area"
+  ],
```

Comment:
I think there are a few extra fields in here?

Reply:
the extra ones appear to be present in all the specialist publisher schemas, including the dfid_research_outputs
https://github.com/alphagov/search-api/blob/3cecbcfb011ef28940e9b66b175da21a4ba4849a/config/schema/elasticsearch_types/dfid_research_output.json#L2

Reply:
Hmm interesting... something to worry about later I think 😆